frontend/src/scenes/data-warehouse/saved_queries/activityDescriptions.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+    ActivityLogItem,
+    defaultDescriber,
+    HumanizedChange,
+    userNameForLogItem,
+} from 'lib/components/ActivityLog/humanizeActivity'
+import { SentenceList } from 'lib/components/ActivityLog/SentenceList'
+
+export function dataWarehouseSavedQueryActivityDescriber(
+    logItem: ActivityLogItem,
+    asNotification?: boolean
+): HumanizedChange {
+    if (logItem.scope != 'DataWarehouseSavedQuery') {
+        console.error('data warehouse saved query describer received a non-data warehouse saved query activity')
```

Comment:
Should this console log be here?

Reply:
Kept it consistent with the other descriptor handlers